### PR TITLE
FCHub submission SG19

### PR DESCRIPTION
**Pull request by minmar94**

### CODE REVIEW

**Comment by kathsherratt** [comment]
Hi Marco, @minmar94 

Thank you for uploading! I can see that the forecasts submitted here are valid. I think the failing check is due to a technical issue on our side (the same error occurred in #1799 ). Please ignore the failing check, we will try to sort this out.

Best wishes,
Kath

**Comment by minmar94** [comment]
Dear Kath,
thank you very much for your kind response. I really appreciate your help,
and hope that the errors could be sorted out.
I'd take the chance to ask if the paper about the ensemble predictions has
been submitted in the end...any news?
Best,
Marco


Il giorno dom 3 apr 2022 alle ore 22:03 Kath Sherratt <
***@***.***> ha scritto:

**Comment by Bisaloo** [comment]
Hi @minmar94, sorry for the confusion. I fixed the bug in our validation framework. It now returns an actual error in your submission.

The `target_end_date` for spatialrichards forecasts L18 to L25 should be shifted one week ahead.

File: data-processed/Statgroup19-spatialrichards/2022-04-04-Statgroup19-spatialrichards.csv [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+2022-04-16,"IT","3 wk ahead inc death","2022-04-04","quantile",260,0.025
+2022-04-23,"IT","4 wk ahead inc death","2022-04-04","quantile",210,0.025
+2022-04-02,"IT","1 wk ahead inc death","2022-04-04","point",852,NA
+2022-04-09,"IT","2 wk ahead inc death","2022-04-04","point",747,NA
+2022-04-16,"IT","3 wk ahead inc death","2022-04-04","point",666,NA
+2022-04-23,"IT","4 wk ahead inc death","2022-04-04","point",600,NA
+2022-04-02,"IT","1 wk ahead inc death","2022-04-04","quantile",1415,0.975
+2022-04-09,"IT","2 wk ahead inc death","2022-04-04","quantile",1419,0.975
+2022-04-16,"IT","3 wk ahead inc death","2022-04-04","quantile",1398,0.975
+2022-04-23,"IT","4 wk ahead inc death","2022-04-04","quantile",1375,0.975
```
Comment (Bisaloo) [inline data-processed/Statgroup19-spatialrichards/2022-04-04-Statgroup19-spatialrichards.csv:25]: The `target_end_date` for forecasts L18 to L25 should be shifted one week ahead.

**Comment by minmar94** [comment]
Dear Hugo,
can I do this right now?

Il giorno lun 4 apr 2022 alle ore 11:38 Hugo Gruson <
***@***.***> ha scritto:

**Comment by Bisaloo** [comment]
@minmar94, as far as I can tell, the problem is still here. If you check L18 of [2022-04-04-Statgroup19-spatialrichards.csv](https://github.com/covid19-forecast-hub-europe/covid19-forecast-hub-europe/pull/1800/files#diff-5726c11729ec41bc5dc7a9c716fb28a43db69f1fbe1a25d3688b8fd07c64ec69), you will notice that you make a forecast in the past (April 2)

**Comment by minmar94** [comment]
I just tried updating the files again, but keep getting the same error.
Could you please upload them for me, I would really appreciate it.
Thank you,
Marco

Il giorno lun 4 apr 2022 alle ore 12:10 Hugo Gruson <
***@***.***> ha scritto:

**Pull request merged**